src/tracks/playabletrack/wavetrack/ui/WaveTrackView.h

```diff
@@ -34,6 +34,8 @@ class AUDACITY_DLL_API WaveTrackSubView : public CommonTrackView
    
    virtual const Type &SubViewType() const = 0;
 
+   virtual void CopyToSubView(WaveTrackSubView *destSubView) const;
```

Comment:
Comment what this is for, and explain that the default implementation does nothing.